src/utils/material.js

```diff
@@ -127,3 +131,9 @@ function handleTextureEvents (el, texture) {
   });
 }
 module.exports.handleTextureEvents = handleTextureEvents;
+
+module.exports.isHLS = function (videoEl) {
+  if (videoEl.type && videoEl.type.toLowerCase() in ['application/x-mpegurl', 'application/vnd.apple.mpegurl']) { return true; }
```

Comment:
arrays don't work like this. you've got to use `includes` or `indexOf` (which has better compatibility across the board). test for this?